Library/Homebrew/download_strategy.rb

```diff
@@ -262,6 +262,10 @@ def initialize(name, resource)
   end
 
   def fetch
+    if @url.include?("https://downloads.sourceforge.net") || @url.include?("https://download.gnome") || @url.include?("https://www.apache.org/dyn/")
```

Comment:
I cut off the country code because I was trying to shorten an already lengthy line. ` Sourceforge `'s could also be cut for the same reason.

I don't mind either way though. We can do either, ultimately, if the maintainers like the idea.